x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -73,3 +73,10 @@ export interface Benchmark {
   agent_policy: Pick<GetAgentPoliciesResponseItem, 'id' | 'name' | 'agents'>;
   rules: CspRulesStatus;
 }
+
+export type {
```

Comment:
hmm.. I don't have a strong opinion about it. 
I just think it is easier to work when all types reside in types index.

Since this is exporting the type out of the schema.. so maybe the naming here is missleading as well.
any suggestions?

Reply:
The way I see it - if we already have these variables exported in this scope (i.e. the plugin scope) we should not re-export it.

It is very standard to export types out of different files, and not necessarily from `types.ts`. `types.ts` exists as a place to put types that need to be shared and don't have a better place to live in, or to avoid cyclic dependencies